README.md

```diff
@@ -57,3 +57,7 @@ Your lab directory must include
 * 3pts Feature Tasks
 * 3pts Tests (free points today)
 * 2pts Documentation
+
+#Documentation
```

Comment:
Good project description, but if I don't know ANYTHING about how to use terminal, how would I even start? Include instructions for how to use the chat server if you want full points for the README.